### ISSUE TITLE
Shared folder names do not appear

### ISSUE THREAD

**Issue by guglovich** [syncthing/syncthing#8390]
- syncthing-1.20.2-1 and Arch
- Firefox 101.0.1 and Chromium 102.0.5005.115
- the names of the folders I have shared on the second PC are not displayed. the names are in plain English
- go to Edit device > Provisioning > "View list of shared folders. folders that I have not exchanged with the names.

![Снимок экрана_2022-06-16_14-59-49](https://user-images.githubusercontent.com/54148165/174112263-ddcf6fa1-3c24-4955-aeca-8d8001d1fecd.png)

**Comment by tomasz1986** [comment]
Duplicate of https://github.com/syncthing/syncthing/issues/8369.
